<launch>
  <!-- This launch file is used for dev purposes -->


  <!-- Load data location parameters -->
  <rosparam command="load" file="$(find avida_ros)/param/phenotypic_plasticity/relaxed_selection_as_limitation/data_locs.yaml" />
  <!-- Load analysis parameters -->
  <rosparam command="load" file="$(find avida_ros)/param/phenotypic_plasticity/relaxed_selection_as_limitation/analysis.yaml" />

  <!-- Treatment 1 Group -->
  <group ns="treatment_1">
    <!-- Load trace server params -->
    <rosparam command="load" file="$(find avida_ros)/param/trace_server.yaml" />
    <!-- Load genotype server parameters -->
    <rosparam command="load" file="$(find avida_ros)/param/genotype_server.yaml" />
    <!-- Launch trace server node for treatment 1 -->
    <node pkg="avida_ros" type="trace_server.py" output="screen" name="trace_server" />
    <!-- Launch genotype server node for treatment 1 -->
    <node pkg="avida_ros" type="genotype_server.py" output="screen" name="genotype_server" />
  </group>

  <!-- Treatment 2 Group -->
  <group ns="treatment_2">
    <!-- Load trace server params -->
    <rosparam command="load" file="$(find avida_ros)/param/trace_server.yaml" />
    <!-- Load genotype server parameters -->
    <rosparam command="load" file="$(find avida_ros)/param/genotype_server.yaml" />
    <!-- Launch trace server node for treatment 2 -->
    <node pkg="avida_ros" type="trace_server.py" output="screen" name="trace_server" />
    <!-- Launch genotype server node for treatment 2 -->
    <node pkg="avida_ros" type="genotype_server.py" output="screen" name="genotype_server" />
  </group>

  <!-- Treatment 3 Group -->
  <group ns="treatment_3">
    <!-- Load trace server params -->
    <rosparam command="load" file="$(find avida_ros)/param/trace_server.yaml" />
    <!-- Load genotype server parameters -->
    <rosparam command="load" file="$(find avida_ros)/param/genotype_server.yaml" />
    <!-- Launch trace server node for treatment 3 -->
    <node pkg="avida_ros" type="trace_server.py" output="screen" name="trace_server" />
    <!-- Launch genotype server node for treatment 2 -->
    <node pkg="avida_ros" type="genotype_server.py" output="screen" name="genotype_server" />
  </group>

  <node pkg="avida_ros" type="pp_rsal_analyze.py" name="RSaL_analysis" output="screen"/>

</launch>
